airflow/providers/microsoft/azure/hooks/container_instance.py

```diff
@@ -177,6 +177,8 @@ def get_logs(self, resource_group: str, name: str, tail: int = 1000) -> list:
         :return: A list of log messages
         """
         logs = self.connection.containers.list_logs(resource_group, name, name, tail=tail)
+        if logs.content is None:
```

Comment:
`get_logs` should return `list | None` :)

Reply:
Thanks, changed return type to list as suggested. Made it [None] and built in a check for the list being longer than 0 and having first index be None to not experience index issues at the start of the run when logs is an empty list - all tested on our setup.